### ISSUE TITLE
Model implementation

### ISSUE THREAD

**Issue by nmpribeiro** [gabordemooij/redbean#479]
Hi there! I'm not the most prolific with php... but... I'm missing something here

```
//'Manufacturer/Models/Model.php'
<?php namespace Manufacturer\Models;
class Model extends \RedBean_SimpleModel{}
?>


//'Manufacturer/Models/User.php'
<?php namespace Manufacturer\Modes;
class User extends Model{}
?>


//'start.php'
define( 'REDBEAN_MODEL_PREFIX', '\\Manufacturer\\Models\\Model' );
//get db... 
$user = new Manufacturer\Models\User();
$user->dispense();
$user->username = 'test';
$user->update();
```

But... yields Call to undefined method TachyonApp\Models\User::dispense()
Wasn't it supposed to be FUSED?

**Comment by nmpribeiro** [comment]
It would be great to see one implementation of this.

**Comment by gabordemooij** [comment]
Instead of $user = new ... try:

     $user = R::dispense('user');

Your $user will now be 'fused' with the User Model.

**Comment by nmpribeiro** [comment]
Yes. 
I've moved $this->bean = R::dispense($this->table); into the model's dispense function (and made it aware of id's for now... I'll expand as I need).

Well, I've created a wrapper 'BaseModel' class that extends nothing, done by myself that acomplished most of general purpose tasks with my beans. Also created myself the idea of 'repositories' but I do see how this must be something done per 'Entity'. RedBeans is not interested in those, since those are project related.
Either way, thanks. Indeed your documentation is enough to understand it. I just purpose a small complete example for others that might not see through.
